tests/cfncluster-release-check.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python
 #
-# Copyright 2018      Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Don't remove the 5 spaces for next year.  It just means more changes next year.

Reply:
ok, I'm going to update the patch